integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -343,6 +343,13 @@
                         params: {
                             inScreen: 'ggumtest' // REQUIRED str Tracking Id
                         }
+                    },
+                    {
+                        bidder: 'wideorbit',
+                        params: {
+                            pbId: 123, // REQUIRED Publisher Id,
+                            pId: 123456 // REQUIRED Placement Id
+                        }

```

Comment:
`pbjs_example_gpt.html` already includes the same wideorbit params further up the page, are these needed or can they be removed?